firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -18,7 +18,9 @@ plugins {
 }
 
 firebaseLibrary {
+    libraryGroup "appdistribution"
```

Comment:
cc: @davidmotson 

Was there a reason appdistro _wasn't_ in its library group, or was it just an oversight?

Reply:
most likely an oversight